src/AgGrid.fs

```diff
@@ -1,194 +1,801 @@
+            | SelectCellEditor
+            | NumberCellEditor
+            | DateCellEditor
+            | DateStringCellEditor
+            | CheckboxCellEditor
+            | LargeTextCellEditor
+            | TextCellEditor
+            | RichSelectCellEditor
+
+            member this.RichCellEditorText = sprintf "ag%O" this
+
+        [<RequireQualifiedAccess>]
+        type RowGroupingDisplayType =
+            | SingleColumn
+            | MultipleColumns
+            | GroupRows
+            | Custom
+
+            member this.RowGroupingDisplayTypeText =
+                match this with
+                | SingleColumn -> "singleColumn"
+                | MultipleColumns -> "multipleColumns"
+                | GroupRows -> "groupRows"
+                | Custom -> "custom"
+
+        [<RequireQualifiedAccess>]
+        type RowGroupPanelShow =
+            | Always
+            | OnlyWhenGrouping
+            | Never
+
+            member this.RowGroupPanelShowText =
+                match this with
+                | Always -> "always"
+                | OnlyWhenGrouping -> "onlyWhenGrouping"
+                | Never -> "never"
+
+        [<RequireQualifiedAccess>]
+        type AggregateFunction =
+            | Sum
+            | Min
+            | Max
+            | Count
+            | Avg
+            | First
+            | Last
+
+            member this.AggregateText = (sprintf "%O" this).ToLower()
+
+        type MenuItemDef = {
+            name: string
+            action: unit -> unit
+            shortcut: string
+            icon: obj //HtmlElement
+        }
+
+        [<RequireQualifiedAccess>]
+        type BuiltInMenuItem =
+            | AutoSizeAll
+            | ExpandAll
+            | ContractAll
+            | Copy
+            | CopyWithHeaders
+            | CopyWithGroupHeaders
+            | Cut
+            | Paste
+            | ResetColumns
+            | Export
+            | CsvExport
+            | ExcelExport
+            | ChartRange
+            | PivotChart
+
+            member this.BuiltInMenuItemText =
+                match this with
+                | AutoSizeAll -> "autoSizeAll"
+                | ExpandAll -> "expandAll"
+                | ContractAll -> "contractAll"
+                | Copy -> "copy"
+                | CopyWithHeaders -> "copyWithHeaders"
+                | CopyWithGroupHeaders -> "copyWithGroupHeaders"
+                | Cut -> "cut"
+                | Paste -> "paste"
+                | ResetColumns -> "resetColumns"
+                | Export -> "export"
+                | CsvExport -> "csvExport"
+                | ExcelExport -> "excelExport"
+                | ChartRange -> "chartRange"
+                | PivotChart -> "pivotChart"
+
+        type MenuItem =
+            | BuiltIn of BuiltInMenuItem
+            | Custom of MenuItemDef
+
+        /// See https://www.ag-grid.com/react-data-grid/grid-interface/#grid-api.
+        [<Erase>]
+        type IGridApi<'row> =
```

Comment:
This type is not referred to anywhere, so we can't access the additional functions (copyToClipboard and pasteFromClipboard). To make that work, we'd need to add new enterprise types referring to it (like `ICellFocusedEvent`), and then new methods that use those types (like `onCellFocused`). Pretty quickly we're duplicating _a lot_ of code, so it might be worth thinking more deeply about that.

To get things moving more quickly for now, we could remove this type, merge the PR and add an issue to address this.